docs/tre-templates/workspace-services/guacamole.md

```diff
@@ -33,3 +33,16 @@ Service Tags:
   make bundle-build DIR=./templates/workspace_services/guacamole
   make bundle-install DIR=./templates/workspace_services/guacamole
   ```
+
+## Guacamole Workspace Service Configuration
+
+On deploying a Guacamole service into a workspace the following properties are exposed.
```

Comment:
```suggestion
When deploying a Guacamole service into a workspace the following properties need to be configured.
```

Reply:
maybe mention that the last property is mandatory and the rest are optional?
you can list the mandatory property first.

Reply:
@eladiw good point.  Will need doing separate to this PR but currently the Client ID is not set as a required property in the template_schema.json file for the Guacamole Service despite having no default.

Reply:
Even if its not marked as mandatory (wrongfully) I guess the doc can still be correct. wdyt

Reply:
Agree. Whether the schema highlights it or not it is still mandatory to successfully deploy the workspace service

Reply:
All suggested changes implemented